nautobot/dcim/views.py

```diff
@@ -199,15 +201,42 @@ def get_selected_objects_parents_name(self, selected_objects):
 #
 
 
-class LocationTypeListView(generic.ObjectListView):
-    queryset = LocationType.objects.all()
-    filterset = filters.LocationTypeFilterSet
-    filterset_form = forms.LocationTypeFilterForm
-    table = tables.LocationTypeTable
-
-
-class LocationTypeView(generic.ObjectView):
+class LocationTypeUIViewSet(
+    view_mixins.ObjectDetailViewMixin,
+    view_mixins.ObjectListViewMixin,
+    view_mixins.ObjectEditViewMixin,
+    view_mixins.ObjectDestroyViewMixin,
+    view_mixins.ObjectBulkDestroyViewMixin,
+    view_mixins.ObjectBulkCreateViewMixin,  # 3.0 TODO: remove this mixin as it's no longer used
+    view_mixins.ObjectChangeLogViewMixin,
+    view_mixins.ObjectNotesViewMixin,
```

Comment:
Can we not just use `NautobotUIViewSet`? Why all the individual mixins?

Reply:
Because of `BulkEdit`, which `LocationType` do not support

Reply:
Let's get an issue opened for adding LocationType bulk-edit support for its `content-types` at least.

Reply:
https://github.com/nautobot/nautobot/issues/6347